src/en/readcomic/src/eu/kanade/tachiyomi/extension/en/readcomic/ReadComic.kt

```diff
@@ -0,0 +1,226 @@
+    override fun latestUpdatesFromElement(element: Element) = SManga.create().apply {
+        val selector = "ul.line-list > li > a.big-link"
+        setUrlWithoutDomain(element.select(selector).attr("href"))
+        title = element.select(selector).text()
+        thumbnail_url = fetchThumbnailURL(element.select(selector).attr("href"))
+    }
+    override fun searchMangaFromElement(element: Element) = SManga.create().apply {
+        setUrlWithoutDomain(element.select("div.dlb-right > a.dlb-title").attr("href"))
+        title = element.select("div.dlb-right > a.dlb-title").text()
+        thumbnail_url = element.select("a.dlb-image > img").attr("src")
+    }
+    private fun fetchThumbnailURL(url: String) = client.newCall(GET(url, headers)).execute().asJsoup().select("div.anime-image > img").attr("src")
+
+    // private fun fetchPagesFromNav(url: String) = client.newCall(GET(url, headers)).execute().asJsoup()
+
+    override fun popularMangaNextPageSelector() = "div.general-nav > a:contains(Next)"
+
+    override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun searchMangaSelector() = "div.dl-box"
+
+    override fun searchMangaNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun mangaDetailsParse(document: Document): SManga {
+        return SManga.create().apply {
+            title = document.select("h1.title").text()
+            thumbnail_url = document.select("div.anime-image > img").attr("src")
+            status = parseStatus(document.select("ul.anime-genres li.status").text())
+            author = document.select("td:contains(Author:) + td").text()
+            description = document.select(".detail-desc-content > p").text()
+            genre = document.select("ul.anime-genres > li > a[href*='genre']").joinToString { it.text() }
+        }
+    }
+
+    private fun parseStatus(element: String): Int = when {
+        element.contains("Completed") -> SManga.COMPLETED
+        element.contains("Ongoing") -> SManga.ONGOING
+        else -> SManga.UNKNOWN
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val document = response.asJsoup()
+        val chapters = ArrayList<SChapter>()
+
+        document.select(chapterListSelector()).forEach {
+            chapters.add(chapterFromElement(it))
+        }
+
+        return chapters
+    }
+
+    override fun chapterListSelector() = "ul.basic-list > li"
+
+    override fun chapterFromElement(element: Element): SChapter {
+        val chapter = SChapter.create()
+        chapter.setUrlWithoutDomain(element.select("a.ch-name").attr("href"))
+        chapter.name = element.select("a.ch-name").text()
+        chapter.date_upload = dateParse(element.select("span").text())
+        return chapter
+    }
+
+    private fun dateParse(dateAsString: String): Long {
+        val date: Date? = SimpleDateFormat("MM/dd/yyyy", Locale.ENGLISH).parse(dateAsString)
+
+        return date?.time ?: 0L
+    }
```

Comment:
make dateformat a class constant and also add try catch;
the Contributing guide has an example implementation